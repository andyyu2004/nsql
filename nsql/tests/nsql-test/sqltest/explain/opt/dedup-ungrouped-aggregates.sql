EXPLAIN
SELECT
    SUM(a),
    SUM(a),
    SUM(a)
FROM (
    VALUES (1),
        (2),
        (3)) AS t (a)
ORDER BY
    SUM(a)
