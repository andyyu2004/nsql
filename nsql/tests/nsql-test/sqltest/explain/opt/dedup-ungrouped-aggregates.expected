 projection (:project @0 :ret, :project @1 :ret, :project @2 :ret)                       
   order by (:project @3 :ret)                                                           
     projection (:project @0 :ret, :project @0 :ret, :project @0 :ret, :project @0 :ret) 
       ungrouped aggregate (sum(:project @0 :ret))                                       
         scan 3 values                                                                   
                                                                                         